standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -1613,6 +1618,30 @@ public void alter_database(final String dbName, final Database newDB) throws TEx
     }
   }
 
+  private boolean isAlterReplSpecific(Database oldDb, Database newDb) {
```

Comment:
Requires a rebase. Seems there is some similar logic in ``isReplicationEventIdUpdate`` see if you can refactor and reuse some part of it